pydy/viz/scene.py

```diff
@@ -24,6 +24,12 @@
     from IPython.html import widgets
     from IPython.display import display, Javascript
 
+    # Support for ipython 2 and ipython 3
+    if IPython.__version__[0] == '2':
```

Comment:
There is one more issue here. The code currently supports not having IPython installed, i.e. IPython is an optional dependency. Either the code needs to take that logic into account or we need to make IPython a hard dependency.

If we go with the former, `display_ipython()` needs to raise an error if IPython is not installed.